code/game/machinery/newscaster.dm

```diff
@@ -165,6 +165,8 @@ var/list/obj/machinery/newscaster/allCasters = list() //Global list that will co
 	var/allow_comments = 1
 	luminosity = 0
 	anchored = 1
+	var/const/articlefont = "Verdana"
+	var/const/signfont = "Times New Roman"
```

Comment:
Should probably be a define, as such:
#define ARTICLEFONT "Verdana"
#define SIGNFONT "Times New Roman"